src/YesSql.Abstractions/ITypeService.cs

```diff
@@ -1,9 +1,16 @@
 using System;
+using System.Collections.Generic;
 
 namespace YesSql
 {
     public interface ITypeService
     {
-         string this[Type t] { get; set; }
+        string this[Type t] { get; set; }
```

Comment:
Maybe only add one more indexer. `Type this[string t]` and remove the other properties from the interface.

Reply:
That doesn't feel right, I think that would add some confusion.  With a single indexer it should be obvious what the intended usage is.  But with two separate indexers, the behavior will not obvious just by looking at it.

Reply:
Convert it to method names if you prefer

Reply:
Looked at it a few different ways.  I think two indexers makes sense as long as only one of them allows setting